### PR TITLE
gh-secret-set: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

The `prompt` for the secret was added in the github-cli v1.10.0, I think the examples were invalid.

### CODE REVIEW

File: pages/common/gh-secret-set.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`gh secret set {{name}} < {{path/to/file}}`
+
+- Set a secret for a specific repository:
+
+`gh secret set {{name}} --body {{value}} --repo {{owner}}/{{repository}}`
+
+- Set an organization secret for specific repositories:
+
+`gh secret set {{name}} --org {{organization}} --repos {{repository1,repository2}}`
```
Comment (CleanMachine1) [inline pages/common/gh-secret-set.md:20]:
```suggestion
`gh secret set {{name}} --org {{organization}} --repos {{repository1,repository2 ...}}`
```

Comment (bl-ue) [reply pages/common/gh-secret-set.md:20]: I think it would be better to put a comma before the ellipses instead of the space, @CleanMachine1.

Comment (CleanMachine1) [reply pages/common/gh-secret-set.md:20]: Edited to reflect your comment

Comment (navarroaxel) [reply pages/common/gh-secret-set.md:20]: the `space` is strange, maybe we should remove it. @bl-ue?

Comment (bl-ue) [reply pages/common/gh-secret-set.md:20]: I meant _instead_ of the space :)

**Review by CleanMachine1 — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]

File: pages/common/gh-secret-set.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`gh secret set {{name}} < {{path/to/file}}`
+
+- Set a secret for a specific repository:
+
+`gh secret set {{name}} --body {{value}} --repo {{owner}}/{{repository}}`
+
+- Set an organization secret for specific repositories:
+
+`gh secret set {{name}} --org {{organization}} --repos {{repository1,repository2,...}}`
```
Comment (marchersimon) [inline pages/common/gh-secret-set.md:20]:
```suggestion
`gh secret set {{name}} --org {{organization}} --repos "{{repository1,repository2,...}}"`
```
Is there a reason you omitted the quotation marks? According to the docs they should be there.

Comment (navarroaxel) [reply pages/common/gh-secret-set.md:20]: In most cases no one adds spaces or `\` to the repository names.

**Review by marchersimon — APPROVED** [review]
Thanks @navarroaxel

**Pull request merged**